libs/tokens/src/hooks/tokens/useNetworkLogo.ts

```diff
@@ -0,0 +1,41 @@
+import { useMemo } from 'react'
+
+import { BaseChainInfo, getChainInfo } from '@cowprotocol/common-const'
+
+import { useBridgeSupportedNetworks } from '../../../../../apps/cowswap-frontend/src/modules/bridge/hooks/useBridgeSupportedNetworks'
+
+export function useNetworkLogo(chainId?: number) {
+  if (!chainId) {
+    return { isLoading: false, logoUrl: undefined }
```

Comment:
It might cause unecessary rerenders, because every time you return a new object.
It's better to cache it in a const.